server/src/main/java/org/opensearch/cluster/awarenesshealth/ClusterAwarenessAttributesHealth.java

```diff
@@ -54,20 +56,20 @@ public ClusterAwarenessAttributesHealth(
 
         // This is the Map which is storing the per attribute value health stats.
         // The key would be the attribute value like rack-1 and the stats would be stored as value.
-        Map<String, ClusterAwarenessAttributeValueHealth> attributesHealth = new HashMap<>();
+        Map<String, List<String>> attributesHealth = new HashMap<>();
```

Comment:
lets rename this as this doesn't consists of health , but list of nodes node. Edit the comment above as well.